build.gradle

```diff
@@ -27,6 +27,9 @@ ext {
 
   slf4jVersion = '2.0.3'
   mockitoVersion = '4.9.0'
+  
+  interlokCommonsOfflineJavadocDir = project.rootDir.getCanonicalPath() + "/interlok-common/build/docs/javadoc"
```

Comment:
I don't think we need this line, and maybe not the one below either.

Reply:
same as the comment above, saw it in interlok-core and thought it was one of the required changes to the build file for junit 5 and java 17. Also confirmed it doesn't need the line below either like you said :)